src/Characters.js

```diff
@@ -1,18 +1,48 @@
 import React, { Component } from 'react';
 import './styles/App.css';
-import CharacterInfoCard from './CharacterInfoCard.js'
+import CharacterInfoCard from './CharacterInfoCard.js';
+import { characters } from './data';
 
 
 class Characters extends Component {
+  constructor() {
+    super();
+    this.state = {
+      data: characters.slice(0, characters.length)
+    }
+  }
+
+
+
+
   render() {
     return (
       <div className="characters-page">
```

Comment:
Looks great. Do you think it is redundant to add the section or can you put the grid on the div className="characters-page". Just a thought, but no edits necessarily needed.

Reply:
I think it would be possible to set up the grid in that way. I added the section to make it a bit easier for styling the card in a certain way.

Reply:
Good catch! That was redundant.